### PR TITLE
fix: bundled installer trigger was not running

### PR DESCRIPTION
**Pull request by mcdurdin**
* The `$_ENV` superglobal was not populated on azure so we did not see the variable. Also renamed and made all caps, probably not necessary but not important.
* The download trigger went to the keyboard id, not the keyboard source path as is required by the build scripts.

### CODE REVIEW

File: _includes/async/keyboard-download.php [hunk 1]
```diff
@@ -150,25 +151,43 @@ function hasNewerBundleVersion($id, $downloads) {
     return $shouldBuild;
   }
 
+  /**
+   * Retrieves the keyboard source path from the keyboard_info api
+   */
+  function getKeyboardSourcePath($id) {
+    // Get the source path
+    $keyboard_info = @file_get_contents("https://api.keyman.com/keyboard/$id");
+    if($keyboard_info === FALSE) {
```
Comment (darcywong00) [inline _includes/async/keyboard-download.php]: If a keyboard isn't found, won't $keyboard_info = 
```
{
    "message": "Keyboard not found"
}
```

Comment (darcywong00) [reply _includes/async/keyboard-download.php]: nvm, I see the status FALSE

File: keyboards/download.php [hunk 2]
```diff
@@ -81,11 +81,18 @@ function validateParameters(&$id, &$platform, &$mode, &$cid, &$target) {
    * Start a background PHP process to do the async work so we don't block the user
    */
   function triggerDownloadBackgroundProcesses($cid, $id, $platform, $mode) {
+    $bearer_token = getenv('TEAMCITY_TOKEN');
+    if($bearer_token === FALSE) {
+      error_log("ERROR: [download.php] TEAMCITY_TOKEN is not configured.");
```
Comment (darcywong00) [inline keyboards/download.php]: Do you want this in a block?
```
if(DEBUG) {
  error_log("ERROR: ...");
}
```

Comment (mcdurdin) [reply keyboards/download.php]: This one is a configuration problem so I would like to be able to see it in the logs if it fails. The others may well be bad input so they just add noise to the log.

**Pull request merged**